### ISSUE TITLE
Safari screenshots generated when using Chrome with --headless

### ISSUE THREAD

**Issue by yelsayd** [zinserjan/wdio-visual-regression-service#81]
I'm running visual regression tests with the following section in my wdio.config.js:
```
  capabilities: [{
    maxInstances: 5,
    browserName: 'chrome',
    chromeOptions: {
      args: headless ? ['--headless', '--disable-gpu', '--window-size=1024,800'] : [],
    },
  }],
```
If and only if `headless` is true, adding the argument to the Chrome process, screenshots get generated for both Chrome and Safari.

**Comment by tnypxl** [comment]
I have the same issue. For some reason `context.browser.name` returns Safari 7.0 when you use the headless flag with chrome.

**Comment by shwetaneelsharma** [comment]
Yes, I too am having this issue. Screenshots get appended with Safari.

**Comment by zinserjan** [comment]
See bestiejs/platform.js#145. 
Until this gets merged and released you could use the browser name of capabilities as a workaround.
